lib/.gemspec.in

```diff
@@ -0,0 +1,21 @@
+# -*- encoding: utf-8 -*-
+
+Gem::Specification.new do |s|
+  s.name = "facter"
+  version = "@LIBFACTER_VERSION_MAJOR@.@LIBFACTER_VERSION_MINOR@.@LIBFACTER_VERSION_PATCH@"
+  mdata = version.match(/(\d+\.\d+\.\d+)/)
+  s.version = mdata ? mdata[1] : version
+
+  s.required_rubygems_version = Gem::Requirement.new(">= 0")
+  s.authors = ["Puppet Labs"]
+  s.date = "2012-08-08"
+  s.description = "You can prove anything with facts!"
+  s.email = "info@puppetlabs.com"
+  s.executables = ["facter"]
+  s.files = ["bin/facter"]
```

Comment:
I don't have a problem with this the way it is, it's as true os Hiera or Puppet's Gemspec.